distribution/sdk/pom.xml

```diff
@@ -17,11 +17,11 @@
 
     <parent>
         <groupId>ddf</groupId>
-        <artifactId>ddf-parent</artifactId>
-        <version>3.0.4-SNAPSHOT</version>
+        <artifactId>distribution</artifactId>
+        <version>2.8.0-SNAPSHOT</version>
     </parent>
 
-    <groupId>ddf.sdk</groupId>
+    <groupId>ddf.distribution</groupId>
     <artifactId>sdk</artifactId>
     <version>2.8.0-SNAPSHOT</version>
```

Comment:
Do you need this 2.8.0-SNAPSHOT version? I would have expected this pom to now inherit this version from distribution/pom.xml.